assets/css/app/components/_observer.scss

```diff
@@ -0,0 +1,38 @@
+.overflow{
+    max-height: 1000px;
+    margin-bottom: 10px;
+    overflow:auto;    
+    -webkit-overflow-scrolling: touch;
+    padding-right: 0px;
+}
+
+
+ul.active_items button:focus{
+    outline: none !important;
+    background-color: $color-gray-100;
+  }
+  
+ul.active_items button:hover{
+outline: none !important;
+background-color: $color-gray-100;
+}
+  
+.node{
+    fill: $color-gray-100;
+    stroke: $color-gray-warm-200;
+    stroke-width:1;
+}
+
+.node:hover{
+    cursor: pointer;
+  }
+  
+.line{
+    stroke: $color-gray-warm-600;    
+    stroke-width:2
+}
+
+.tree{
+    position: relative; 
+    top: 20px;
+}
```

Comment:
Looks awesome!

I have three small comments regarding css:

1. I think we should scope the css (nest tem all under the `.observer` class)

Otherwise I think there is a good chance of observer styles impacting different components.

(example of scoping - for the `Code Field` component, where everything is nested under `.code_field`: https://github.com/phoenixframework/phoenix_live_dashboard/blob/master/assets/css/app/components/_code_field.scss )

2. In CSS we use two spaces for indentation and add a space before opening the `{`.

3. What do you think about changing the name of the from `observer` to something else (like `supervision-tree`)? 
When I read `observer`, I think about a more general set of tool, not only the supervision tree display.

Reply:
Thanks for comments.
Yes, that name sounds better. I will make those changes